Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM quay.io/cdis/python:3.7-alpine as base
+
+FROM base as builder
+RUN apk add --no-cache --virtual .build-deps gcc musl-dev libffi-dev openssl-dev make postgresql-dev git curl vim
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+COPY . /src/
+WORKDIR /src
+RUN python -m venv /env && . /env/bin/activate && $HOME/.poetry/bin/poetry install --no-dev --no-interaction
```

Comment:
I love this pattern of having a root `/env` in which we run `poetry` 🥇  I saw that in [metadata-service](https://github.com/uc-cdis/metadata-service/blob/master/Dockerfile) and thought it made a lot of sense.

What are the expectations we have around community member usage though? Do we want community users to be able to run the Docker container locally and use tools like `pytest` to run tests? Is that an exercise we leave to end users? I was wondering since `--no-dev` would exclude testing tools, and since we don't bundle `$HOME/.poetry` in the second stage build they'd have to install poetry from scratch too.

Reply:
I see, this dockerfile was strictly written for deployment, I guess we could have a separate dockerfile for running tests, to keep this one simple - do you have any recommendations?